xdsl/interpreters/pdl_interp.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
is this necessary?

Reply:
no, I must've left that during development, I removed it now.